cmake/ceed.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=${prefix}/include
+libdir=${prefix}/lib
+
+Name: CEED
+Description: Code for Efficient Extensible Discretization
+Version: 0.12.0
+Cflags: -I${includedir}
+Libs: -L${libdir} -lceed
+Libs.private: @CEED_LIBS_PRIVATE@
```

Comment:
I think we can unify with what is now `ceed.pc.template`, adopting the CMake syntax if that is required.